### DISCUSSION TITLE
"backup" is almost 50% larger than source...

### DISCUSSION THREAD

**Discussion by Zamana** [psy0rz/zfs_autobackup#160] [General]
Hi!

I'm testing zfs_autobackup, and after my 2 initials try, I'm with this situation:

My source data has these sizes:

![image](https://user-images.githubusercontent.com/6900313/182662372-09affd90-521e-42bd-a84a-fbc786be2417.png)

And the backup data now has these sizes:

![image](https://user-images.githubusercontent.com/6900313/182662521-cabfa9bd-3c1a-46b4-9529-25341c941fa4.png)

"seafile" dataset increased just a little bit, and that's ok, but why "yumi" dataset increased almost 50%?

The destination dataset "probox/autobackup" was empty before.

**Comment by psy0rz** [comment]
Do you perhaps use different compression settings on the source and target?

**Reply by Zamana** [reply]
Maybe.

Anyway, I did the backup again to another pool, and the sizes are equal now.

Thanks.
Regards.

**Comment by psy0rz** [comment]
ok thanks :)

**Comment by alaricljs** [comment]
This is often a symptom of different ashift or different pool layout (RZ1/2/3) between the pools.
